src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    /* eslint-disable */
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+    /* eslint-enable */
```

Comment:
Recommendation: try not to disable `eslint` and write the code on several lines
```suggestion
    get info() {
      return 
        `name: ${this.name}, `
        + `chip version: ${this.version}, '
        + 'wheels: ${this.wheels}`;
    },
```

Reply:
Yeah it can be done like that but for this line it just looks weird to me to write this in separate lines. But sure I try not to disable `eslint` normally.